Documentation/CMake.md

```diff
@@ -59,21 +59,28 @@ endif()
 ## Add an entry point
 
 You must include a source file in your test executable target with a
-`@main` entry point. The following example uses the SwiftPM entry point:
+`@main` entry point.
 
 ```swift
-import Testing
+import Foundation
+@_spi(ForToolsIntegrationOnly) import Testing
 
 @main struct Runner {
-    static func main() async {
-        await Testing.__swiftPMEntryPoint() as Never
+    static func main() async throws {
+        let configurationJSON: UnsafeRawBufferPointer? = nil
```

Comment:
You can leave this line out and just pass `nil` if you're not actually going to use it. :)

Reply:
Not quite, even when just passing nil Swift 6 gives me an isolation warning. marking this as `nonisolated(unsafe)` makes that go away.

Reply:
Calloo callay. Okay, that's fine then.